### DISCUSSION TITLE
Provide Split View membership indicator for tabs

### DISCUSSION THREAD

**Discussion by n-xlkt** [zen-browser/desktop#2244] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

# Actual Result
## Overview
There is no indication in the tab list that a tab is part of a Split View. The lack of this creates accessibility issues in the following scenarios.

### Scenario 1
- *Given* that a user has created one or more Split Views,
- *When* the user has switched to a tab that is not a member of a Split View,
- *Then* it becomes unclear in the tab list which tabs are members of a Split View.

### Scenario 2
- *Given* that the user has duplicate tabs open,
- *When* at least one of the duplicate tabs gains membership into a Split View,
- *Then* it becomes unclear in the tab list which of the duplicate tabs is a member of said Split View.

### Scenario 3
- *Given* that the user is visiting the same domain across at least 2 tabs,
    - *And*  the Sidebar Width has been toggled to narrow, 
- *When* at least one of the tabs visiting the same domain gains membership into a Split View,
- *Then* it becomes unclear in the tab list which one of the tabs visiting the same domain is a member of said Split View.

# Expected Result
There should be a clear affordance provided in the tab list that indicates tab membership in a Split View.

## References and Recommendations
Arc handles this by visually merging tabs that are members of the same split-view in the tab list, but there might be less heavy-handed options (e.g., reordering relevant tabs so that they're adjacent to one another in the tab list and using a UI element to make it clear that they're currently part of a single split-view).

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.7

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1891/provide-split-view-membership-indicator-for-tabs">ZEN-1891 Provide Split View membership indicator for tabs</a></p>

**Comment by nanoraptor** [comment]
Bump this! This one is essential if the browser is using split view.

**Comment by mr-cheffy** [comment]
>  I am not filing an enhancement request.

please, next time read the checkboxes, it takes a huge time for me to migrate the issues
